### PR TITLE
Switch to kusd and kumo

### PR DESCRIPTION
**Pull request by wollooo**
see https://github.com/kumodao/borrowprot/issues/34

### CODE REVIEW

**Comment by wollooo** [comment]
I switched LUSD to KUSD in the entrie rroject. Is this a problem @udosson ?

**Comment by wollooo** [comment]
I messed up to create a new branch for the switch to KUMO. So in this branch you can find the switch from LQTY to KUMO.

**Comment by udosson** [comment]
I'm still reviewing but I see some files still named `*liquity*` like `docs/sdk/lib-base.liquitystorebasestate.totalstakedunitokens.md`
Could we change them to `*kumo*` as well?

**Review by udosson — CHANGES_REQUESTED** [review]
@wollooo This PR is awesome! Your regex skills rock!
Could you please make the small adjustments found as comments and in our discord chat?
Also, it would be great if you could change all `*liquity*`, `*lqty*`, and `*lusd*` filenames to the corresponding `kumo` names.
I'm looking forward to merging your PR.

File: docs/sdk/lib-base.liquidationdetails.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ export interface LiquidationDetails
 |  Property | Type | Description |
 |  --- | --- | --- |
 |  [collateralGasCompensation](./lib-base.liquidationdetails.collateralgascompensation.md) | [Decimal](./lib-base.decimal.md) | Amount of native currency (e.g. Ether) paid to the liquidator as gas compensation. |
+|  [KUSDGasCompensation](./lib-base.liquidationdetails.kusdgascompensation.md) | [Decimal](./lib-base.decimal.md) | Amount of KUSD paid to the liquidator as gas compensation. |
```
[the remark this answers is not in the document]
Comment (wollooo) [reply docs/sdk/lib-base.liquidationdetails.md:20]: yes it is automatically generated

File: docs/sdk/lib-base.liquitystorebasestate.kusdbalance.md [hunk 2]
```diff
@@ -1,13 +1,13 @@
 <!-- Do not edit this file. It is automatically generated by API Documenter. -->
 
-[Home](./index.md) &gt; [@liquity/lib-base](./lib-base.md) &gt; [LiquityStoreBaseState](./lib-base.liquitystorebasestate.md) &gt; [lqtyBalance](./lib-base.liquitystorebasestate.lqtybalance.md)
+[Home](./index.md) &gt; [@liquity/lib-base](./lib-base.md) &gt; [LiquityStoreBaseState](./lib-base.liquitystorebasestate.md) &gt; [kusdBalance](./lib-base.liquitystorebasestate.kusdbalance.md)
 
-## LiquityStoreBaseState.lqtyBalance property
+## LiquityStoreBaseState.kusdBalance property
```
[the remark this answers is not in the document]
Comment (wollooo) [reply docs/sdk/lib-base.liquitystorebasestate.kusdbalance.md:5]: Everything looks right to me. I think git has mapped the changes incorrectly. There is always one file for KUSD and one for KUMO. See here:

- [kumobalance.md](https://github.com/wollooo/borrowprot/blob/switch-to-kusd/docs/sdk/lib-base.liquitystorebasestate.kumobalance.md)
- [kusdbalance.md](https://github.com/wollooo/borrowprot/blob/switch-to-kusd/docs/sdk/lib-base.liquitystorebasestate.kusdbalance.md)

File: docs/sdk/lib-base.liquitystorebasestate.kusdbalance.md [hunk 3]
```diff
@@ -1,13 +1,13 @@
...
 
-User's LQTY token balance.
+User's KUSD token balance.
```
[the remark this answers is not in the document]
Comment (wollooo) [reply docs/sdk/lib-base.liquitystorebasestate.kusdbalance.md:7]: https://github.com/kumodao/borrowprot/pull/66#discussion_r853255253

File: docs/sdk/lib-base.liquitystorebasestate.kusdbalance.md [hunk 4]
```diff
@@ -1,13 +1,13 @@
...
 
 <b>Signature:</b>
 
 ```typescript
-lqtyBalance: Decimal;
+kusdBalance: Decimal;
```
[the remark this answers is not in the document]
Comment (wollooo) [reply docs/sdk/lib-base.liquitystorebasestate.kusdbalance.md:12]: https://github.com/kumodao/borrowprot/pull/66#discussion_r853255253

File: docs/sdk/lib-ethers.ethersliquity.getkumobalance.md [hunk 5]
```diff
@@ -1,15 +1,15 @@
 <!-- Do not edit this file. It is automatically generated by API Documenter. -->
 
-[Home](./index.md) &gt; [@liquity/lib-ethers](./lib-ethers.md) &gt; [EthersLiquity](./lib-ethers.ethersliquity.md) &gt; [getLUSDBalance](./lib-ethers.ethersliquity.getlusdbalance.md)
+[Home](./index.md) &gt; [@liquity/lib-ethers](./lib-ethers.md) &gt; [EthersLiquity](./lib-ethers.ethersliquity.md) &gt; [getKUMOBalance](./lib-ethers.ethersliquity.getkumobalance.md)
```
[the remark this answers is not in the document]
Comment (wollooo) [reply docs/sdk/lib-ethers.ethersliquity.getkumobalance.md:3]: similar to https://github.com/kumodao/borrowprot/pull/66#discussion_r853255253

**Review by udosson — APPROVED** [review]
@wollooo thank you so much for this amazing contribution. Kudos!!!

**Pull request merged**